app/views/student_quizzes/take_quiz.html.erb

```diff
@@ -14,6 +14,11 @@
 
   <%= hidden_field_tag :questionnaire_id, "#{@questionnaire.id}" %>
   <%= hidden_field_tag :assignment_id, "#{params[:assignment_id]}" %>
-  <%= render :partial => 'quiz_form', :locals => { :questionnaire_id => @questionnaire.id,:display=>"take" } %>
```

Comment:
Do a search in the project first. My guess is that it is not used anywhere after your refactoring.